### ISSUE TITLE
🐛 in foot-extra, nvim buffer still shows after quitting. Screen does not clear. 

### ISSUE THREAD

**Issue by felixniemeyer** [manjaro-sway/manjaro-sway#819]
### branch

stable

### Architecture

x86_64

### Description

When I open a file in nvim and quit, the file contents will still be shown. 

I'd like to *not* see the file contents anymore after quitting nvim, because 
1. I'd like to see all previous commands
2. I can better find my terminal windows, when they look like nvim only if nvim is actually running. 

### Reproduction steps

```bash
nvim .zshrc # or some other text file
# then in vim: 
:q
```
=> I see still see the nvim file content lines except the last line, which is a new prompt.

### Screenshots

![image](https://github.com/manjaro-sway/manjaro-sway/assets/5720176/8c27c316-10aa-4d0b-8003-610331afed83)

**Comment by felixniemeyer** [comment]
If I run `export TERM=foot` the vim screen clears fine when quitting: 

![image](https://github.com/manjaro-sway/manjaro-sway/assets/5720176/a586ab3a-6195-48ab-b76f-e38dbfce2168)

**Comment by felixniemeyer** [comment]
Setting `TERM` to `foot` instead of `foot-extra` is my workaround right now. 

But I read in the Arch Wiki: "Do not set the TERM variable from your ~/.bashrc or ~/.bash_profile or similar file. The terminal itself should report the correct TERM to the system so that the proper terminfo file will be used. "

So I hope there will be a clean solution. 

Let me know if I can try something!

**Comment by boredland** [comment]
Hm, maybe this is due to me using lazyvim, but for me this looks just fine:

[recording_20240623-110723.webm](https://github.com/manjaro-sway/manjaro-sway/assets/4662748/70a6e75c-bc66-4c7b-ba6d-6a6565a030cc)

**Comment by felixniemeyer** [comment]
that looks fine indeed.

I just tried disabling my nvim config (mv .config/nvim/ .config/nvim2) and the issue persists with TERM=foot-extra. 

If I also disable my foot config (mv .config/foot/ .config/foot2) then the issue goes away! 

Commenting out the first line fixes the issue with the remaining nvim lines. 
```
include=/usr/share/sway/templates/foot.ini
```
Ah okay, no wonder, this file sets `term=foot-extra` and we already know setting `term=foot` fixes the issue. 

To sum up: even with default (=no custom) nvim and foot config the issue persists. 

I just tried commenting out .zshrc, didn't help either. 

Can you confirm your TERM env variable is `foot-extra`?

**Comment by boredland** [comment]
> Can you confirm your TERM env variable is foot-extra?

Ya, it is foot-extra. How confusing.

**Comment by boredland** [comment]
Sadly not reproducible for me..
